tests/CitizenProject/CitizenProjectFactoryTest.php

```diff
@@ -65,5 +68,6 @@ public function testCreateCitizenProjectFromCitizenProjectCreationCommand()
         $this->assertSame($problemDescription, $citizenProject->getProblemDescription());
         $this->assertSame($proposedSolution, $citizenProject->getProposedSolution());
         $this->assertSame($requiredMeans, $citizenProject->getRequiredMeans());
+        $this->assertCount(1, $citizenProject->getSkills());
```

Comment:
Would be better with something like `assertSame([$skill], $citizenProject->getSkills()->toArray())`;

Reply:
ok, but why ? I would understand...

Reply:
Because here it is just the size of the returned value that is tested.
`assertSame` is more precise. It would strictly compare the type of collection and the data of the collection itself.
It's not a big deal but it's better 😉